front/src/applications/operationalStudies/views/Project.tsx

```diff
@@ -55,6 +63,24 @@ export default function Project() {
     [urlProjectId]
   );
 
+  const {
+    selectedItemIds: selectedStudyIds,
+    setSelectedItemIds: setSelectedStudyIds,
+    items: studiesList,
+    setItems: setStudiesList,
+    toggleSelection: toggleStudySelection,
+    deleteItems,
+  } = useMultiSelection<StudyWithScenarios>((studyId) => {
+    if (projectId !== undefined) {
+      deleteStudyV2({ projectId, studyId });
+    }
```

Comment:
We don't need this test anymore here since the check already done inside the `handleDeleteStudy`. To avoid ts error due to the type, you can use [Non null assertion operator](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-) in like this `deleteStudyV2({ projectId:projectId!, studyId });`.